tests/test_sca_models.py

```diff
@@ -0,0 +1,194 @@
+                                    ],
+                                }
+                            ],
+                        }
+                    ],
+                },
+            ),
+            (
+                SCAScanDiffOutput,
+                {
+                    "scanned_files": ["toto"],
+                    "added_vulns": [
+                        {
+                            "location": "toto",
+                            "package_vulns": [
+                                {
+                                    "package_full_name": "toto",
+                                    "version": "1.2.4",
+                                    "ecosystem": "pypi",
+                                    "dependency_type": "direct",
+                                    "vulns": [
+                                        {
+                                            "severity": "LOW",
+                                            "summary": "toto",
+                                            "identifier": "foo",
+                                            "cve_ids": ["CVE_1", "CVE_2"],
+                                            "created_at": None,
+                                            "fixed_version": "1.2.3",
+                                        }
+                                    ],
+                                }
+                            ],
+                        }
+                    ],
+                    "removed_vulns": [
+                        {
+                            "location": "toto",
+                            "package_vulns": [
+                                {
+                                    "package_full_name": "toto",
+                                    "version": "1.2.4",
+                                    "ecosystem": "pypi",
+                                    "dependency_type": "direct",
+                                    "vulns": [
+                                        {
+                                            "severity": "LOW",
+                                            "summary": "toto",
+                                            "identifier": "foo",
+                                            "cve_ids": ["CVE_1", "CVE_2"],
+                                            "created_at": None,
+                                            "fixed_version": "1.2.3",
+                                        }
+                                    ],
+                                }
+                            ],
+                        }
+                    ],
+                },
+            ),
+        ],
+    )
+    def test_schema_loads(self, expected_klass, instance_data):
+        """
+        GIVEN the right kwargs and an extra field in dict format
+        WHEN loading using the schema
+        THEN the extra field is not taken into account
+            AND the result should be an instance of the expected class
```

Comment:
nit: it's odd to indent `AND` (I see it's been done in two other tests in this repo, I think it's a mistake)

```suggestion
        AND the result should be an instance of the expected class
```

Reply:
Indeed, the mistake has been done once and copy/paste everywhere. fixed